showcase.json

```diff
@@ -4099,6 +5184,48 @@
     "github": "https://github.com/ASPPIBRA-DAO",
     "twitter": "https://x.com/ASPPIBRA_ORG",
     "youtube": "https://www.youtube.com/@ASPPIBRA-DAO/videos",
-    "submittableId": ""
+    "submitta,bleId": ""
+  },
+  {
+    "id": "Blueband",
+    "name": "Blueband",
+    "description": "A vector database built based on Vectra, a local vector database for Node.js.",
+    "logo": "/img/showcase/blueband_logo.png",
+    "display": "Normal",
+    "oneLiner": "A vector database built based on Vectra, a local vector database for Node.js",
+    "tags": [
+      "AI"
+    ],
+    "usesInternetIdentity": false,
+    "github": "https://github.com/highfeast/ic-use-blueband-db"
+  },
+  {
+    "id": "Cipher-AI-Vault",
+    "name": "Cipher AI Vault",
+    "description": "Azle-based proof-of-concept designed integrating in-memory VectorDB and LLM.",
+    "logo": "/img/showcase/cipherAI_logo.jpeg",
+    "display": "Normal",
+    "oneLiner": "Azle-based proof-of-concept designed integrating in-memory VectorDB and LLM.",
+    "tags": [
+      "AI"
+    ],
+    "usesInternetIdentity": true,
+    "github": "https://github.com/supaIC/Cipher-AI-Vault",
+    "website": "https://qehbq-rqaaa-aaaan-ql2iq-cai.icp0.io/"
+  },
+  {
+    "id": "zcloak",
+    "name": "zCloak Network",
+    "description": "Defend trust and privacy in the age of AI by offering rapid, cost-effective, and secure zero-knowledge proof verification services for blockchains.",
+    "logo": "/img/showcase/zCloak_logo.svg",
+    "display": "Normal",
+    "oneLiner": "DID and ZKP Infrastructure for Web3",
+    "tags": [
+      "AI",
+      "Tools / Infrastructure"
+    ],
+    "usesInternetIdentity": true,
```

Comment:
which product uses internet identity? I am struggling struggling to find a login with II

Reply:
Ah yes, I meant to say false here. I updated it.